skymap_scanner/server/start_scan.py

```diff
@@ -126,6 +126,22 @@ def __init__(
 
         self.omgeo = g_frame["I3Geometry"].omgeo
 
+        # Attributes necessary for the pointed scan with splinempe_pointed.
+        # If the reco isn't splinempe_pointed, they don't affect the scanner.
+        self.online_ra_dec = None
+        self.ang_dist = 3.5
+
+        if self.reco.get_use_online_ra_dec():
+            particle_name_possibilities = ["OnlineL2_SplineMPE", "l2_online_SplineMPE"]
+            for particle_name in particle_name_possibilities:
+                if particle_name in p_frame.keys():
+                    online_dir = p_frame[particle_name].dir
+                    self.online_ra_dec = astro.dir_to_equa(
+                        online_dir.zenith,
+                        online_dir.azimuth,
+                        self.event_header.start_time.mod_julian_day_double
+                    )
+
```

Comment:
Can you move this to a dedicated method in the reco class, and then return only the needed values? This will help keep these common functions brief.